src/components/ui/admin/settings/listSlides/item.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Flex, Input, AutoComplete, Checkbox } from 'antd';
+import { useMotionValue, Reorder } from 'framer-motion';
+import { Close } from '~components/ui/icons/close';
+import { useRaisedShadow } from './hooks/useRaisedShadow';
+const { TextArea } = Input;
+
+export const Item = ({
```

Comment:
Could we have better naming and call the file Item? And please call the file exactly like component Item